Magento_Theme/templates/html/pager.phtml

```diff
@@ -44,22 +44,18 @@
                         <?php endforeach; ?>
                     </select>
                 </div>
-
-                <span class="dashboard-toolbar__amount-info">
-                    <?php /* @escapeNotVerified */ echo __('per page') ?>
-                </span>
             </div>
         <?php endif; ?>
 
-        <?php if ($block->getLastPageNum()>1): ?>
-            <div class="pager dashboard-toolbar__pager grid__column">
+        <?php if ($block->getLastPageNum() > 1): ?>
+            <div class="pager toolbar__pager">
                 <ul class="pager__list" aria-labelledby="paging-label">
                     <?php if (!$block->isFirstPage()): ?>
                         <li class="pager__item">
                             <?php $text = $block->getAnchorTextForPrevious() ? $block->getAnchorTextForPrevious() : '';?>
 
-                            <a class="<?php /* @escapeNotVerified */ echo $text ? 'link ' : 'action '?> pager__link previous" 
-                               href="<?php /* @escapeNotVerified */ echo $block->getPreviousPageUrl() ?>" 
+                            <a class="<?php /* @escapeNotVerified */ echo $text ? 'link ' : 'action '?> pager__link previous"
```

Comment:
attrs to new line and ise shorhand instead of echo

Reply:
splited and added shorhand